homeassistant/config.py

```diff
@@ -355,6 +356,7 @@ def _validate_currency(data: Any) -> Any:
                 _no_duplicate_auth_mfa_module,
             ),
             vol.Optional(CONF_MEDIA_DIRS): cv.schema_with_slug_keys(vol.IsDir()),
+            vol.Remove(CONF_LEGACY_TEMPLATES): cv.boolean,
```

Comment:
Should we test that it doesn't blow up and we can still startup ok when this is present?

Reply:
I just added a test for it